datatypes/src/main/java/org/hyperledger/besu/datatypes/GWei.java

```diff
@@ -111,7 +111,7 @@ public static GWei fromHexString(final String str) {
    * @return Wei
    */
   public Wei getAsWei() {
-    return Wei.of(getAsBigInteger().multiply(BigInteger.TEN.pow(9)));
+    return Wei.of(getAsBigInteger().multiply(BigInteger.valueOf(1_000_000_000L)));
```

Comment:
nitpick: We might even bring `BigInteger.valueOf(1_000_000_000L)` out as a constant.

Reply:
Done